man/systemd.network.xml

```diff
@@ -2287,6 +2287,56 @@
       </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[TrafficControlQueueingDiscipline] Section Options</title>
+    <para>The <literal>[TrafficControlQueueingDiscipline]</literal> section manages the Traffic control. It can be used
+    to configure the kernel packet scheduler and simulate packet delay and loss for UDP or TCP applications,
+    or limit the bandwidth usage of a particular service to simulate internet connections.</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>Parent=</varname></term>
+        <listitem>
+          <para>Specifies the parent Queueing Discipline (qdisc). Takes one of <literal>root</literal> or <literal>clsact</literal>.
```

Comment:
Now I'd like to test this PR. What does `clsact` mean? How to test that value?

Reply:
Are you able to test the `root` ? The clsact qdisc works on ingress or egress. See https://lwn.net/Articles/671458/

```
[TrafficControlQueueingDiscipline]
Parent=clsact

```

```
tc qdisc                                                                                                                                                                                 ─╯
qdisc clsact ffff: dev veth99 parent ffff:fff1 
```

Also you can replace the `root` with `clsact` and test with the example I have already given

Reply:
Thanks!